docs/assets/sample_expectation_suite_output.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+div.terminal {
+  color: white;
+  background-color: black;
+}
+div.terminal pre {
+  background-color: black;
+  color: white;
+  padding: 20px;
+  overflow: auto
+}
+</style>
+<body>
+```scala mdoc:passthrough
```

Comment:
I take it this is where you've struggled ? Is Laika somehow able to run this snippet via a call to mdoc ?

Reply:
When this passthrough is in a markdown document, Laika hangs. Mdoc doesn't have an issue. By moving it to an html document laika ignores the file.

Reply:
Mmm that's not ideal :/ , the point of this mdoc invocation is to exemplify the output of the framework. Not that it's critical to the docs but it's kinda nice to have.

Probs worth pinging some of the laïka maintainers for advice ?

Reply:
This was a problem with the ansi to html transformation making invalid html, fixed by making the html output valid.

see 8b1bca779916e43fedeb96f92398bb18f9cce0db

Reply:
@CJSmith-0141, would you mind just uploading a screenshot of the microsite on the PR description ?